frontend/app/views/spree/users/show.html.erb

```diff
@@ -0,0 +1,48 @@
+<h1><%= accurate_title %></h1>
+
+<div data-hook="account_summary" class="account-summary">
+  <dl id="user-info">
+    <dt><%= Spree.t(:email) %></dt>
+    <dd>
+      <%= @user.email %>
+      <% if spree.respond_to? :edit_account_path %>
+        (<%= link_to Spree.t(:edit), spree.edit_account_path %>)
+      <% end %>
+    </dd>
+  </dl>
+</div>
+
+<div data-hook="account_my_orders" class="account-my-orders">
+
+  <h3><%= Spree.t(:my_orders) %></h3>
+  <% if @orders.present? %>
+    <table class="order-summary">
+      <thead>
+      <tr>
+        <th class="order-number"><%= I18n.t(:number, scope: 'activerecord.attributes.spree/order') %></th>
+        <th class="order-date"><%= Spree.t(:date) %></th>
+        <th class="order-status"><%= Spree.t(:status) %></th>
+        <th class="order-payment-state"><%= Spree.t(:payment_state) %></th>
+        <th class="order-shipment-state"><%= Spree.t(:shipment_state) %></th>
+        <th class="order-total"><%= Spree.t(:total) %></th>
+      </tr>
+      </thead>
+      <tbody>
+      <% @orders.each do |order| %>
+        <tr class="<%= cycle('even', 'odd') %>">
+          <td class="order-number"><%= link_to order.number, order_url(order) %></td>
+          <td class="order-date"><%= l order.completed_at.to_date %></td>
+          <td class="order-status"><%= Spree.t("order_state.#{order.state}").titleize %></td>
```

Comment:
Please do not use `titleize` here. As this is cosmetic change that should be done solely in CSS

Reply:
Also do not use `Spree.t`